react/week2/to_do_list/src/index.js

```diff
@@ -0,0 +1,129 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import "./index.css";
+
+const todos = [
+  {
+    id: 0,
+    description: "Get out of bed",
+    color: "brown",
+    deadline: "2020-06-08",
+    done: false,
+    Visibility: true
+  },
+  {
+    id: 1,
+    description: "Brush teeth",
+    color: "red",
+    deadline: "2020-06-08",
+    done: false,
+    Visibility: true
+  },
+  {
+    id: 2,
+    description: "Eat breakfast",
+    color: "orange",
+    deadline: "2020-06-08",
+    done: true,
+    Visibility: true
+  }
+];
+
+function getRandomColor() {
+  var letters = "0123456789ABCDEF";
+  var color = "#";
+  for (var i = 0; i < 6; i++) {
+    color += letters[Math.floor(Math.random() * 16)];
+  }
+  return color;
+}
+
+class TodoList extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { renderTrigger: true };
+  }
```

Comment:
In the version of react we're using, you can omit the constructor and super, it's done behind the scenes, so you could re-write this like: 
```suggestion
    state = { renderTrigger: true };
```
You can read more about it in the [docs](https://reactjs.org/docs/react-component.html#constructor), however I will admit, they are a bit confusing to understand.